web/pgadmin/dashboard/static/js/Dashboard.jsx

```diff
@@ -6,13 +6,13 @@
 // This software is released under the PostgreSQL Licence
 //
 //////////////////////////////////////////////////////////////
-import React, { useEffect, useMemo, useState } from 'react';
+import React, { useEffect, useMemo, useState, Fragment } from 'react';
 import { styled } from '@mui/material/styles';
 import gettext from 'sources/gettext';
 import PropTypes from 'prop-types';
 import getApiInstance from 'sources/api_instance';
 import PgTable from 'sources/components/PgTable';
-import { InputCheckbox } from '../../../static/js/components/FormComponents';
+import { InputCheckbox, FormInputSwitch, FormInputToggle } from '../../../static/js/components/FormComponents';
```

Comment:
![image](https://github.com/pgadmin-org/pgadmin4/assets/31855755/84f1fce3-4bbf-4dc4-8fd3-2facfe8b2e68)

Reply:
Can we move these suggestions to some separate Issue?

Reply:
No it is not in CM.

Reply:
It is in the CM as well as the outer one. <img width="1410" alt="Screenshot 2024-07-02 at 11 01 03 AM" src="https://github.com/pgadmin-org/pgadmin4/assets/14345545/d340d164-ab03-4adb-a4f2-b6290ac77def">

The outer one is in all the Dashboard tabs, so that needs to be fixed.

Reply:
I think then you just need to adjust the height of your content and override some CSS. Dashboard tabs are generic to fit all requirements, its not a bug. You can tweak as per your need.

Reply:
Done.